figure_generation/manuscript_figure_generation.py

```diff
@@ -0,0 +1,566 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Figure Generation for Manuscript
+
+# In[1]:
+
+
+from pathlib import Path
+from IPython.display import Image, display, SVG
+
+from cairosvg import svg2png
+from lxml import etree
+import numpy as np
+import svgutils.transform as sg
+
+
+# ## Figure 1
+
+# In[2]:
+
+
+corpora_comparison_path = Path("../biorxiv/corpora_comparison/output/")
+
+
+# In[3]:
+
+
+panel_one = (
+    sg.fromfile(f"{str(corpora_comparison_path)}/svg_files/corpora_kl_divergence.svg")
+)
+
+# Convert pt units to pixel units
+# Vince's tutorial FTW
```

Comment:
@vincerubinetti you've reached the Greenelab milestone of being mentioned in David's code